src/Groups/gsets.jl

```diff
@@ -351,8 +362,8 @@ julia> length(orbit(Omega, 1))
 """
 function orbit(Omega::GSetByElements{<:GAPGroup, S}, omega::S) where S
     G = acting_group(Omega)
-    acts = GapObj(gens(G))
-    gfun = GapObj(action_function(Omega))
+    acts = GapObj(gens(G); recursive=true)
```

Comment:
```suggestion
    acts = [GapObj(g) for in gens(G)]
```
but even this will fail if `g` cannot be converted to a GAP object...